subprojects/packagefiles/indicators/meson.build

```diff
@@ -0,0 +1,16 @@
+project(
+    'indicators',
+    'cpp',
+    version : '2.2',
+    license : 'MIT'
+)
+
+install_headers([
+    'single_include/indicators/indicators.hpp',
+], subdir: 'indicators')
+
+includes = include_directories(['include'])
```

Comment:
This should be `single_include` too I think, otherwise projects that uses indicators as subproject won't include same header than when using installed indicators.